assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -13,6 +13,70 @@ resource "aws_route53_record" "etcds" {
   records = [aws_instance.controllers[count.index].private_ip]
 }
 
+# IAM Policy
+resource "aws_iam_instance_profile" "csi-driver-instance-profile" {
```

Comment:
IMO there is no need to include the resource type in its name. This is just redundant info. `aws_iam_instance_profile.csi-driver` is IMO clearer and more concise than `aws_iam_instance_profile.csi-driver-instance-profile`. Same for other similar occurrences.